[IPP] waiting jobs always returns -NaN in OH 2.5.0.M5

<!-- Provide a general summary of the issue in the *Title* above -->
<!-- If the issue is related to a binding, please include its short name in -->
<!-- square brackets in the title - Example: "[astro] My issue..." -->
All channels like waiting_jobs on IPP printers return -NaN with OH 2.5.0.M5. With 2.4.0 everything works well.

<!-- Important: Please contact the openHAB community forum for questions or -->
<!-- for configuration and usage guidance: https://community.openhab.org -->

<!-- Feel free to delete any comment lines in the template (starting with "<!--") -->

## Expected Behavior
Waiting jobs should return the real printer's waiting jobs value. 

## Current Behavior
<!-- If describing a bug, tell us what happens instead of the expected behavior -->
<!-- Include related log information (preferably debug level) and related configs -->
<!-- Use a file attachment for log and config information longer than a few lines -->
<!-- Enclose multi-line log/code snippets with ``` on new lines for proper formatting -->
<!-- If suggesting a change/improvement, explain the difference from current behavior -->
<!-- For improvements, discuss at community.openhab.org first and include link to topic -->
Waiting jobs always returns -NaN
![image](https://user-images.githubusercontent.com/8273791/69832603-97564e00-122f-11ea-9fa4-927c847437a1.png)


## Possible Solution
<!-- Not obligatory, but suggest a fix/reason for the bug, -->
<!-- or ideas how to implement the addition or change -->
No solution found other the doing an downgrade to OH 2.4.0 release.
I have tried to delete the think / items and recreate it. But this does not help.

## Steps to Reproduce (for Bugs)
<!-- Provide a link to a live example, or an unambiguous set of steps to -->
<!-- reproduce this bug. Include code to reproduce, if relevant -->
1. Add a printer thing
2. Connect an item to waiting jobs channel
3. Add a printer job and observe the waiting jobs item

## Your Environment
<!-- Include as many relevant details about the environment you experienced the bug 
OH 2.5.0.M5
Linux openHABianPi 4.19.66-v7+ #1253 SMP Thu Aug 15 11:49:46 BST 2019 armv7l

After updating the thing settings, I can see the following error in the events.log

2019-11-30 11:51:48.857 [WARN ] [mmon.WrappedScheduledExecutorService] - Scheduled runnable ended with an exception: 
java.lang.NoClassDefFoundError: org/simpleframework/xml/Serializer
	at ch.ethz.vppserver.ippclient.IppAttributeProviderFactory.createIppAttributeProvider(IppAttributeProviderFactory.java:5) ~[?:?]
	at ch.ethz.vppserver.ippclient.IppResponse.<init>(IppResponse.java:54) ~[?:?]
	at org.cups4j.operations.IppOperation.sendRequest(IppOperation.java:220) ~[?:?]
	at org.cups4j.operations.IppOperation.sendRequest(IppOperation.java:131) ~[?:?]
	at org.cups4j.operations.IppOperation.request(IppOperation.java:67) ~[?:?]
	at org.cups4j.operations.ipp.IppGetJobsOperation.getPrintJobs(IppGetJobsOperation.java:119) ~[?:?]
	at org.cups4j.CupsPrinter.getJobs(CupsPrinter.java:330) ~[?:?]
	at org.openhab.binding.ipp.internal.handler.IppPrinterHandler.onDeviceStateChanged(IppPrinterHandler.java:131) ~[?:?]
	at org.openhab.binding.ipp.internal.handler.IppPrinterHandler.lambda$0(IppPrinterHandler.java:110) ~[?:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_222]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[?:1.8.0_222]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[?:1.8.0_222]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[?:1.8.0_222]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_222]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_222]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_222]
Caused by: java.lang.ClassNotFoundException: org.simpleframework.xml.Serializer cannot be found by org.openhab.binding.ipp_2.5.0.M5
	at org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:484) ~[org.eclipse.osgi-3.12.100.jar:?]
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:395) ~[org.eclipse.osgi-3.12.100.jar:?]
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:387) ~[org.eclipse.osgi-3.12.100.jar:?]
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.loadClass(ModuleClassLoader.java:150) ~[org.eclipse.osgi-3.12.100.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357) ~[?:1.8.0_222]
	... 16 more

--- a/bundles/org.openhab.binding.ipp/pom.xml
+++ b/bundles/org.openhab.binding.ipp/pom.xml
@@ -26,6 +26,12 @@
       <scope>compile</scope>
     </dependency>
     <dependency>
+        <groupId>org.simpleframework</groupId>
+        <artifactId>simple-xml</artifactId>
+        <version>2.7.1</version>
+        <scope>compile</scope>
+    </dependency>
+    <dependency>
       <groupId>commons-io</groupId>
       <artifactId>commons-io</artifactId>
       <version>2.6</version>

--- a/bundles/org.openhab.binding.ipp/pom.xml
+++ b/bundles/org.openhab.binding.ipp/pom.xml
@@ -14,8 +14,7 @@
   <name>openHAB Add-ons :: Bundles :: IPP Binding</name>
 
   <properties>
-    <dep.noembedding>httpcore-osgi,httpclient-osgi,commons-io</dep.noembedding>
-    <bnd.importpackage>!org.simpleframework.xml.*</bnd.importpackage>
+    <bnd.importpackage>!org.simpleframework.xml.*,!org.cups4j,!net.sf.ehcache.*,!net.spy.*</bnd.importpackage>
   </properties>
 
   <dependencies>
@@ -39,7 +38,7 @@
     <dependency>
       <groupId>commons-io</groupId>
       <artifactId>commons-io</artifactId>
-      <version>2.6</version>
+      <version>1.4</version>
       <scope>compile</scope>
     </dependency>
     <dependency>

--- a/bundles/org.openhab.binding.ipp/pom.xml
+++ b/bundles/org.openhab.binding.ipp/pom.xml
@@ -32,6 +32,11 @@
         <scope>compile</scope>
     </dependency>
     <dependency>
+        <groupId>xmlpull</groupId>
+        <artifactId>xmlpull</artifactId>
+        <version>1.1.3.1</version>
+    </dependency>
+    <dependency>
       <groupId>commons-io</groupId>
       <artifactId>commons-io</artifactId>
       <version>2.6</version>

--- a/bundles/org.openhab.binding.ipp/src/main/feature/feature.xml
+++ b/bundles/org.openhab.binding.ipp/src/main/feature/feature.xml
@@ -5,9 +5,6 @@
     <feature name="openhab-binding-ipp" description="IPP Binding" version="${project.version}">
         <feature>openhab-runtime-base</feature>
         <feature>openhab-transport-mdns</feature>
-        <bundle dependency="true">mvn:org.apache.httpcomponents/httpcore-osgi/4.4.4</bundle>
-        <bundle dependency="true">mvn:org.apache.httpcomponents/httpclient-osgi/4.5.3</bundle>
-        <bundle dependency="true">mvn:commons-io/commons-io/2.6</bundle>
         <bundle start-level="80">mvn:org.openhab.addons.bundles/org.openhab.binding.ipp/${project.version}</bundle>
     </feature>
 </features>

